src/container/cli.c

```diff
@@ -956,6 +975,10 @@ dc_cont_open(tse_task_t *task)
 		dc_task_set_priv(task, cont);
 	}
 
+	/* DAOS_COO_RO_MDSTATS introduced since 2.4 */
```

Comment:
```suggestion
	/* DAOS_COO_RO_MDSTATS introduced since 2.4 (protocol version 7) */
```

very minor, only if you need to update the patch for some other reason